packages/neutron-sdk/src/interchain_queries/register_queries.rs

```diff
@@ -135,35 +94,28 @@ pub fn new_register_delegator_delegations_query_msg(
 /// * **update_period** is used to say how often the query must be updated.
 /// * **min_height** is used to set min height for query (by default = 0).
 pub fn new_register_transfers_query_msg(
-    deps: DepsMut<InterchainQueries>,
-    env: Env,
     connection_id: String,
     recipient: String,
     update_period: u64,
     min_height: Option<u128>,
 ) -> NeutronResult<NeutronMsg> {
-    let mut query_data: Vec<TransactionFilterItem> = vec![TransactionFilterItem {
```

Comment:
And I think it would be pretty convenient if `TransactionFilters::new()` method could accept initial filter items, thus users don't need to call `.push()` when they know initial filter items like it was here.

Reply:
That will require `new()` to return an error as well. Are we okay with that?

Reply:
Personally I think it's ok. But if it's pretty hard to implement and this is completely unidiomatic, let's leave it as it is

Reply:
it is not hard to implement, it just looks weird.

what do you think about approach 3 I have described before?

Reply:
I think TryFrom may be implemented